tachyon/math/finite_fields/generator/generator_util.cc

```diff
@@ -14,6 +14,15 @@
 
 namespace tachyon::math {
 
+bool IsBinaryField(const mpz_class& m) {
+  mpz_class binary_modulus(2);
+  for (int i = 0; i < 8; ++i) {
```

Comment:
Though nitpicky, I think changing this to i <  4, 7, 8 for the commits of up to 8, 64, 128 might prevent confusion.